sdks/python/README.md

```diff
@@ -255,215 +256,212 @@ Some interesting classes to navigate to:
+ | 'read' >> beam.Read(
+    beam.io.TextFileSource('gs://dataflow-samples/shakespeare/kinglear.txt'))
+ | 'split' >> beam.FlatMap(lambda x: re.findall(r'\w+', x))
+ | 'count words' >> beam.combiners.Count.PerElement()
+ | 'save' >> beam.Write(beam.io.TextFileSink('./word_count')))
 p.run()
 ```
 
 ### Counting words with GroupByKey
 
-Here we use `GroupByKey` to count the words.
-This is a somewhat forced example of `GroupByKey`; normally one would use
-the transform `df.combiners.Count.PerElement`, as in the previous example.
-The example also shows the use of a wild-card in specifying the text file
-source.
-
+This is a somewhat forced example of `GroupByKey` to achieve the same
+functionality of the previous example without using
+`beam.combiners.Count.PerElement`. It demonstrates also the use of a
+wildcard to specify the text file source.
 ```python
 import re
-import google.cloud.dataflow as df
-p = df.Pipeline('DirectPipelineRunner')
-class MyCountTransform(df.PTransform):
+import apache_beam as beam
+p = beam.Pipeline('DirectPipelineRunner')
+class MyCountTransform(beam.PTransform):
   def apply(self, pcoll):
     return (pcoll
-    | df.Map('one word', lambda w: (w, 1))
-    # GroupByKey accepts a PCollection of (w, 1) and
-    # outputs a PCollection of (w, (1, 1, ...))
-    | df.GroupByKey('group words')
-    | df.Map('count words', lambda (word, counts): (word, len(counts))))
+            | 'one word' >> beam.Map(lambda word: (word, 1))
+            # GroupByKey accepts a PCollection of (word, 1) elements and
+            # outputs a PCollection of (word, [1, 1, ...])
+            | 'group words' >> beam.GroupByKey()
+            | 'count words' >> beam.Map(lambda (word, counts): (word, len(counts))))
 
 (p
- | df.Read('read', df.io.TextFileSource('./names*'))
- | df.FlatMap('split', lambda x: re.findall(r'\w+', x))
+ | 'read' >> beam.Read(beam.io.TextFileSource('./names*'))
+ | 'split' >> beam.FlatMap(lambda x: re.findall(r'\w+', x))
  | MyCountTransform()
- | df.Write('write', df.io.TextFileSink('./results')))
+ | 'write' >> beam.Write(beam.io.TextFileSink('./word_count')))
 p.run()
 ```
 
 ### Type hints
 
-In some cases, you can improve the efficiency of the data encoding by providing
-type hints.  For example:
+In some cases, providing type hints can improve the efficiency
+of the data encoding.
 
 ```python
-import google.cloud.dataflow as df
-from google.cloud.dataflow.typehints import typehints
-p = df.Pipeline('DirectPipelineRunner')
+import apache_beam as beam
+from apache_beam.typehints import typehints
+p = beam.Pipeline('DirectPipelineRunner')
 (p
- | df.Read('A', df.io.TextFileSource('./names'))
- | df.Map('B1', lambda x: (x, 1)).with_output_types(typehints.KV[str, int])
- | df.GroupByKey('GBK')
- | df.Write('C', df.io.TextFileSink('./results')))
+ | 'read' >> beam.Read(beam.io.TextFileSource('./names'))
+ | 'add types' >> beam.Map(lambda x: (x, 1)).with_output_types(typehints.KV[unicode, int])
```

Comment:
Great catch! I must say I have no explanation. I don't remember changing it (that is why I didn't look for it in the diff), not sure what happened. Just removed it